ons_alpha/core/models/snippets.py

```diff
@@ -99,3 +102,25 @@ class Meta:
 
     def __str__(self):
         return str(self.name)
+
+
+@register_snippet
+class Chart(index.Indexed, PreviewableMixin, models.Model):
```

Comment:
Could you please add it to `BulletinStoryBlock` and update the bulletin template to have the `include_django` include if we have a chart block?